app/code/Magento/Email/Controller/Adminhtml/Email/Template/Save.php

```diff
@@ -1,15 +1,62 @@
 <?php
 /**
- *
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
 namespace Magento\Email\Controller\Adminhtml\Email\Template;
 
+use Exception;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\Session;
+use Magento\Email\Controller\Adminhtml\Email\Template;
+use Magento\Email\Model\ResourceModel\Template as TemplateResource;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\App\TemplateTypesInterface;
+use Magento\Framework\Registry;
+use Magento\Framework\Stdlib\DateTime\DateTime;
 
-class Save extends \Magento\Email\Controller\Adminhtml\Email\Template
+/**
+ * Save Controller
+ */
+class Save extends Template implements HttpPostActionInterface
 {
+    /**
+     * @var DateTime
+     */
+    private $dateTime;
+
+    /**
+     * @var TemplateResource
+     */
+    private $templateResource;
+
+    /**
+     * @var Session
+     */
+    private $backendSession;
+
+    /**
+     * Save constructor
+     *
+     * @param Context $context
+     * @param Registry $coreRegistry
+     * @param DateTime $dateTime
+     * @param TemplateResource $templateResource
+     * @param Session $backendSession
+     */
+    public function __construct(
+        Context $context,
+        Registry $coreRegistry,
+        DateTime $dateTime,
```

Comment:
The new `$dateTime`, `$templateResource `, `$backendSession` constructor parameters should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter) (section *Adding a constructor parameter*) for more details.